shared/menubar/index.desktop.tsx

```diff
@@ -381,8 +398,8 @@ const BadgeIcon = ({tab, countMap, openApp}) => {
   return (
     <Kb.Box style={{...Styles.desktopStyles.clickable, position: 'relative'}}>
       <Kb.Icon
-        color={Styles.globalColors.blueDarker}
-        hoverColor={Styles.globalColors.white}
+        color={Styles.isDarkMode() ? Styles.globalColors.black_50OrBlack_60 : Styles.globalColors.blueDarker}
+        hoverColor={Styles.globalColors.whiteOrWhite}
         onClick={() => openApp(tab)}
         sizeType="Big"
         style={styles.navIcons}
```

Comment:
the `borderBottomColor` for `arrowTick` should match the dark grey used for the menubar header

Reply:
doh! good catch fixed